src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -106,37 +114,43 @@ public ExportPanel(TerminationFuture terminationFuture) {
     txtExportDirectory.setFocusable(false);
     txtExportDirectory.setEditable(false);
     txtExportDirectory.setColumns(10);
+    txtExportDirectory.setText(PREFERENCES.get(EXPORT_DIR_PREF, ""));
 
     btnChooseExportDirectory = new JButton("Choose...");
-    btnChooseExportDirectory.addActionListener(__ -> chooseLocation(directory(
-        this,
-        fileFrom(txtExportDirectory),
-        f -> f.exists() && f.isDirectory() && !isUnderBriefcaseFolder(f) && !isUnderODKFolder(f),
-        "Exclude Briefcase & ODK directories"
-    ), txtExportDirectory));
+    btnChooseExportDirectory.addActionListener(__ -> getExportDirectoryChooser().choose().ifPresent(file -> {
+      txtExportDirectory.setText(file.getAbsolutePath());
+      updateExportButton();
+      PREFERENCES.put(EXPORT_DIR_PREF, file.getAbsolutePath());
+    }));
 
     JLabel lblPemPrivateKey = new JLabel("PEM Private Key File:");
 
     pemPrivateKeyFilePath = new JTextField();
     pemPrivateKeyFilePath.setFocusable(false);
     pemPrivateKeyFilePath.setEditable(false);
     pemPrivateKeyFilePath.setColumns(10);
+    pemPrivateKeyFilePath.setText(PREFERENCES.get(PEM_FILE_PREF, ""));
 
     btnPemFileChooseButton = new JButton("Choose...");
-    btnPemFileChooseButton.addActionListener(__ -> chooseLocation(file(
-        this,
-        fileFrom(pemPrivateKeyFilePath)
-    ), pemPrivateKeyFilePath));
+    btnPemFileChooseButton.addActionListener(__ -> getPemFileChooser().choose().ifPresent(file -> {
+      pemPrivateKeyFilePath.setText(file.getAbsolutePath());
+      updateExportButton();
+      PREFERENCES.put(PEM_FILE_PREF, file.getAbsolutePath());
+    }));
 
     JLabel lblDateFrom = new JLabel("Start Date (inclusive):");
     JLabel lblDateTo = new JLabel("End Date (exclusive):");
 
     pickStartDate = createDatePicker();
+    pickStartDate.setDate(PREFERENCES.get(EXPORT_START_DATE_PREF, LocalDate::parse, null));
     pickStartDate.addDateChangeListener(__ -> updateExportButton());
     pickStartDate.addDateChangeListener(__ -> validateDate(pickStartDate));
+    pickStartDate.addDateChangeListener(event -> PREFERENCES.put(EXPORT_START_DATE_PREF, Optional.ofNullable(event.getNewDate()).map(ld -> ld.format(ISO_DATE)).orElse(null)));
```

Comment:
How about some wrapping here? These really wide lines are hard to read on smaller screens (e.g. my laptop) or when arranging multiple windows horizontally. Even with a wide screen I think it's difficult to have to scan horizontally for a long time.